mmv1/products/compute/RouterNat.yaml

```diff
@@ -128,7 +140,7 @@ properties:
       function: 'verify.ValidateRFC1035Name(2, 63)'
   - !ruby/object:Api::Type::Enum
     name: 'natIpAllocateOption'
-    required: true
+    required: false
```

Comment:
If this field is no longer required, what is the default behaviour for an empty field?

Reply:
I would say not sending the field through the request. For example: if you try to set this field while using the "type=PRIVATE" the api return an error with the following message: "Error creating RouterNat: googleapi: Error 400: Invalid value for field 'resource.nats[0].natIpAllocateOption': 'MANUAL_ONLY'. Setting natIpAllocateOption is unsupported with type=PRIVATE."